src/d3-graph.ts

```diff
@@ -112,34 +115,34 @@ export default class D3Graph {
    * @param {Object} callbacks Object containing callbacks
    * @returns {undefined}
    */
-  _renderNodes(data, callbacks) {
+  private _renderNodes(data: GraphData, callbacks: any): void {
```

Comment:
Is there any meaning to adding "_" before renderNodes?

Reply:
tldr - a code style thing. help keep tabs on private fields/methods, differentiate them from public stuff

being explicit with names of private functions. it's borrowed for languages like Python which dont prevent access to private's of a class (there are no privates actually). I find it keep track of usage of things inside a class -- you know what the scope of that method/field is even without looking at it's deceleration.